n_cells: 6
max_cell_voltage: 4.2
min_cell_voltage: 3.6
bat_capacity_s: 600